scripts/do-release.sh

```diff
@@ -232,6 +242,10 @@ function release_all() {
     for project in ${PROJECTS[*]}; do
         create_project_release "$project"
     done
+
+    if [[ "${release['pre-release']}" != "true" ]]; then
+        release_bundle
```

Comment:
```suggestion
        release_bundle || errors=$((errors+1))
```

Should probably increase error counter to make sense.